docs/adr/0004-editing-microcopy.md

```diff
@@ -43,7 +43,7 @@ Although we could use translations to extract all of the strings in the applicat
    * **Link and button labels** are not extracted. We think link and button labels are unlikely to change, and extracting them made the application tests harder to read by obfuscating some of the crucial steps in the test with translation keys.
    * **Publishing component strings** are not extracted. This ensures we are able to migrate these components to the [govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components) repo, which wouldn't be able to access our local translations.
    * **Big guidance** is extracted into it's own Markdown files and stored alongside the corresponding HTML page that shows it. For example, the guidance for creating a new document is stored in `app/views/new_document/guidance.md`.
-   * **Domain data** that's static is extracted into custom YAML files. This application has two static models (DocumentTypeSchema and SupertypeSchema) that encapsulate domain concepts where the data is part of the application. We think domain data - whether it's a backend setting or a label - should be defined in one place so it can change together. In this application, we've chosen to store the backing files for the two static models in `app/formats`.
+   * **Domain data** that's static is stored in a number of custom YAML files. This application has two static models (DocumentTypeSchema and SupertypeSchema) that encapsulate domain concepts where the data is part of the application. We have split up domain data based on whether it's used in a backend setting or as a label for use in the frontend'. The former is stored in `app/formats` whilst the latter is in `config/locales/en`.
```

Comment:
Almost! This ADR was out-of-date already: we don't store the config in `app/formats` anymore. Maybe just lose that bit?

Reply:
oops should have spotted this, sorry! Will just remove the last sentence